.github/workflows/ci-pr.yml

```diff
@@ -60,3 +60,8 @@ jobs:
       uses: coverallsapp/github-action@v1.0.1
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Upload Dist
+      uses: actions/upload-artifact@v1
+      with:
+        name: dist
+        path: dist
```

Comment:
Does this apply to all sub-packages too, or only core?

Reply:
just to core, unfortunately. Also I believe the URLs expire after 5 minutes :/